config/locales/doorkeeper.fr.yml

```diff
@@ -139,5 +139,5 @@ fr:
       write:media: téléverser des fichiers-média
       write:mutes: silencier des gens et des conversations
```

Comment:
*faire taire* or *rendre muet* instead of *silencier* ?

Reply:
Same thing for `read:mutes` (line 127). Maybe *voir les personnes que vous avez rendu muet*

Reply:
The term used everywhere else is "masquer". I don't know why it's not the case here.
I will fix that, thanks.